pkg/metrics/testing/delivery.go

```diff
@@ -81,16 +83,39 @@ func (e ExpectDelivery) Expect200(t *testing.T, trigger string) {
 
 func (e ExpectDelivery) Verify(t *testing.T) {
 	t.Helper()
-	e.verifyProcessing(t)
-	e.verifyDelivery(t, "event_count")
-	e.verifyDelivery(t, "event_dispatch_latencies")
+	var err error
+	timeout := time.After(2 * time.Second)
```

Comment:
How was this timeout decided? Any evidence that it's long enough?

Reply:
This was chosen arbitrarily, it should probably never take more than a few microseconds for the opencensus worker to update metrics.

Reply:
Optional nit, add a comment saying that this value is arbitrary so future readers know it has no magic significance.